lib/active_model/serializer/adapter/json.rb

```diff
@@ -9,12 +9,6 @@ def serializable_hash(options = nil)
           options ||= {}
           { root => Attributes.new(serializer).serializable_hash(options) }
         end
-
-        private
-
-        def fragment_cache(cached_hash, non_cached_hash)
-          ActiveModel::Serializer::Adapter::Json::FragmentCache.new.fragment_cache(cached_hash, non_cached_hash)
-        end
```

Comment:
Let's stay on the safe side for now (i.e. keeping the `fragment_cache` method), and remove it in a subsequent PR if it turns out it's already defined by something else.